x-pack/legacy/plugins/apm/public/components/shared/Links/apm/TransactionOverviewLink.tsx

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import React from 'react';
+import { APMLink } from './APMLink';
+import { useUrlParams } from '../../../../hooks/useUrlParams';
+import { pickKeys } from '../../../../utils/pickKeys';
+
+interface Props {
+  serviceName: string;
+  children: React.ReactNode;
+}
+
+const TransactionOverviewLink = ({ serviceName, children }: Props) => {
```

Comment:
I can't decide whether it should be possible to override (some of) the `urlParams` like:
```js
const TransactionOverviewLink = ({ serviceName, transactionType,  children }: Props) => {
```

What do you think?

Reply:
do we have any use case for it right now? (not sure if we should open this up already if we don't have any use for it yet)

Reply:
Good point. Let's leave it as-is 👍